### DISCUSSION TITLE
Nokia 2330c-2 (S40v5 FP1 Lite)

### DISCUSSION THREAD

**Discussion by dimon495** [mahomaps/mm-v1#27] [Tested devices]
Не работают сетевые функции, нужны правки в коде для добавки порта :80 к HTTP запросам
![photo_2023-05-25_23-12-45](https://github.com/mahomaps/mm-v1/assets/76395065/cec20489-32fe-4c8b-b5b2-4a2229dd9ae2)
![photo_2023-05-25_23-12-45 (2)](https://github.com/mahomaps/mm-v1/assets/76395065/14075b6d-52fa-456d-ba06-dac9feb490d9)
![photo_2023-05-25_23-12-45 (3)](https://github.com/mahomaps/mm-v1/assets/76395065/ed4e24ab-e13f-4dbf-84d5-94aaac741236)

**Comment by Computershik73** [comment]
Предположу, что либо не настроен APN, либо в URL отсутствует порт. Типичный прикол явы на S40.
UPD: увидел текст, теперь понятно.

**Comment by Feodor0090** [comment]
Вопрос с портами закрыт, _должно быть получше_ (с) в ближайшем релизе
